Sprint1.py

```diff
@@ -1,10 +1,7 @@
-oceny = open("C:\\Users\\kdabek\\Desktop\\UpSkill - Python\\dane_wejsciowe\\sprint1\\grades1.txt", 'r').read()
+oceny = open("C:\\Users\\kdabek\\Desktop\\UpSkill - Python\\dane_wejsciowe\\sprint1\\grades1.txt", 'r')

```

Comment:
instead of hardcoded path try to use https://realpython.com/python-pathlib/ (there is a lot of knowledge at this tutorial, do not be scared, only small subset is needed like: `pathlib.Path('test.md').read_text()`)